plugin/src/main/java/com/denizenscript/denizen/scripts/containers/core/ItemScriptHelper.java

```diff
@@ -162,7 +164,7 @@ public static void registerShapedRecipe(ItemScriptContainer container, ItemStack
                 ingredients.add(items);
             }
         }
-        NamespacedKey key = new NamespacedKey("denizen", internalId);
+        NamespacedKey key = new NamespacedKey(Denizen.getInstance(), internalId);

```

Comment:
this one should also be put back how it was

Reply:
Done